<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="inpImageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>551</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="parseOutTextBrowser">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>371</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>400</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="queryFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="isletmeAdiLabel">
       <property name="text">
        <string>İşlet. Ad</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="isletmeAdiLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tarihLabel">
       <property name="text">
        <string>Tarihi</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="tarihLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="araButton">
       <property name="text">
        <string>Ara</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filePathVBoxLayout">
     <item>
      <widget class="QLabel" name="filePathLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Dosya yolu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filePathLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="filePathButton">
       <property name="text">
        <string>Dosyayı Aç</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="fisTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>551</width>
      <height>191</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>İşletme Adı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tarih</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fiş No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Toplam</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KDV</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="urunlerTableWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Fiyatı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KDV</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ürün Adı</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>510</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="fisAraFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fisAraLabel">
       <property name="text">
        <string>Fiş No   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fisLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="fisAraButton">
       <property name="text">
        <string>Fiş Ara</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>120</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Resmi Çözümle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Verileri Ekle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
